### PR TITLE
Follow-up the PR #3082, etc

### PR DESCRIPTION
**Pull request by ghost**
Follow-up the PR #3082
Follow-up the commit 143ad74db70a363d977a2ef67b00cf22252eb1d8: unnoticed updates
Fix for commit b0f5ba02327c034c7f28c0a88211700aaf3086a4: Z offset: -front [of the nozzle] +behind -> Y offset: -front [of the nozzle] +behind
Standardize the writing method: Z-probe, z-probe -> Z Probe
Adjust spacing

### CODE REVIEW

File: Marlin/example_configurations/Hephestos/Configuration.h [hunk 1]
```diff
@@ -78,7 +78,7 @@ Here are some standard links for getting your machine calibrated:
 #define CUSTOM_MACHINE_NAME "HEPHESTOS"
 
 // Added for BQ
-#define SOURCE_CODE_URL "http://www.bq.com/gb/downloads-prusa-i3-hephestos.html"
+#define DEFAULT_SOURCE_URL "http://www.bq.com/gb/downloads-prusa-i3-hephestos.html"
```
Comment (thinkyhead) [inline Marlin/example_configurations/Hephestos/Configuration.h]: Actually, the original line is correct. The default source URL is based on the `MOTHERBOARD` selection, then configurations may override it with `SOURCE_CODE_URL`.

File: Marlin/example_configurations/WITBOX/Configuration.h [hunk 2]
```diff
@@ -78,7 +78,7 @@ Here are some standard links for getting your machine calibrated:
 #define CUSTOM_MACHINE_NAME "WITBOX"
 
 // Added for BQ
-#define SOURCE_CODE_URL "http://www.bq.com/gb/downloads-witbox.html"
+#define DEFAULT_SOURCE_URL "http://www.bq.com/gb/downloads-witbox.html"
```
Comment (thinkyhead) [inline Marlin/example_configurations/WITBOX/Configuration.h]: same here.

**Comment by thinkyhead** [comment]
Other than the errant change to `SOURCE_CODE_URL` it looks good. Fix it up and it can be merged.

**Comment by ghost** [comment]
I've fixed those.

**Comment by thinkyhead** [comment]
Looks like it now needs a `git rebase RCBugFix` – sorry about that.

**Comment by ghost** [comment]
I think that I was able to rebase, but unnecesary pull-requests stuck again. 
I restart PR If you order.

**Comment by ghost** [comment]
I restart PR, #3206.

**Comment by thinkyhead** [comment]
It's always better to just rebase if you can. It leaves fewer notifications to follow up.

You should always do both a regular `git rebase` to bring the commits up to the end of `RCBugFix`, and then a `git rebase -i` so you can clean up your commits and make them pretty. You can keep on adding new patch-up commits and using `git rebase -i RCBugFix` and `git push -f` to use "fixup" on them right up until the PR is merged. But the first three steps below are vital to be sure you're rebasing onto the latest code:

```
git fetch upstream
git checkout RCBugFix
git merge upstream/RCBugFix
git checkout patch-1
git rebase RCBugFix
git rebase -i RCBugFix
git push -f
```

**Pull request closed without merging**